scripts/pylintrc

```diff
@@ -28,6 +28,7 @@ disable=
     too-many-arguments,
     too-many-branches,
     too-many-instance-attributes,
+    too-many-lines,
```

Comment:
Are you sure we want this global setting?

Reply:
At least until the the test suite has been refactored by @Marenz. I did not want to refactor this file just to eliminate this warning. We could create a ticket to remember to let go of `too-many-lines` in pylintrc again. What do you think?

Reply:
I think you can annotate individual files and even functions.

Reply:
didn't think of being able to do that on a global file level. done.